src/expr/src/scalar/mod.rs

```diff
@@ -890,6 +895,111 @@ impl MirScalarExpr {
                 }
             },
         );
+
+        fn reduce_list_create_list_index_literal(mut exprs: Vec<MirScalarExpr>) -> MirScalarExpr {
```

Comment:
Could you add a `#region` around this and the other associated list functions? (Search this file for examples) Also can you move these below the other reduce helper methods?

Reply:
Done